### DISCUSSION TITLE
🌋誰說github儲存庫只能放code!目前兩個最火的儲存庫~ 我想要發PR成為貢獻者(誤)

### DISCUSSION THREAD

**Discussion by JunTingLin** [1101-GS4719-C/Python-Community-GS4719-C#492] [General]
# 🌋誰說github儲存庫只能放code!目前兩個最火的儲存庫~ ~~我想要發PR成為貢獻者~~

# 前言
上週我修的一門課，助教來跟我們分享開源專案，其中github是必不可少，github除了可以讓我們放程式碼在儲存庫外，當然還可放一些筆記、圖片等等，在此分享兩個走偏但卻是很火的github儲存庫，給大家博君一笑😁

# 兩個長歪🥓的儲存庫
1. [HowToCook程序員如何在家做飯指南🍽](https://github.com/Anduin2017/HowToCook)
![image](https://user-images.githubusercontent.com/92431095/159118637-f1d2243d-b574-45d0-9635-44cc5a12a525.png)

之前疫情，有程式設計師腦動大開，關在家除了coding，最重要的莫過於吃，版主分享自己做菜的菜譜，並邀請大家發PR，merge到此

2. [Dress好耶是女裝👗](https://github.com/komeiji-satori/Dress)
![image](https://user-images.githubusercontent.com/92431095/159118528-11f3ba0d-9d12-46c4-954c-73693766951a.png)

這個....我就不知道是在幹嘛了😅，但版主是希望本項目幫助您學習克隆倉庫、創建分支、推送和拉取提交、發出拉取請求的全過程，從而對 Git/GitHub 的使用有一個很好的了解。聲明一下啊，本人無任何癖好，上面兩個均為助教跟我們說的

# 我要如何跟上最近很夯的儲存庫
[github Trending](https://github.com/trending)
![image](https://user-images.githubusercontent.com/92431095/159118555-28a30664-1d50-405d-a126-066face30ead.png)

# 如何成文開源專案的貢獻者
據我所知假如你覺得某個專案(儲存庫)很有趣，想 ~~佔為己有~~ 為他的專案修改或是加些東西，可以folk回來自己的帳戶，在自己這邊修改完後就可以發PR給原作者，假如對方收下了(他覺得你的很棒)，就merge到對方那邊，成為他專案的貢獻者啦😎

# 結語
github作為世界最大的男性交友平台，據統計有98%的使用者均為男生👦。這邊想要分享即使**不是男生、不會程式、不會coding**，還是可以拿github來當社群交流分享。

# PS
附上學長綠油油的圖片一張~看到整個下巴都掉了吧🙀
![20220311_114953](https://user-images.githubusercontent.com/92431095/159118609-7cb4c5c9-d16a-4eba-b949-6cbe8c067e90.jpg)

**Comment by ghost** [comment]
原來還有這種事，長姿勢了

**Comment by sean1325418** [comment]
放code變成放食譜也太可愛。不過原本就沒規定只能放code了

**Comment by xkllkx** [comment]
果然github就是工程師的社群平台，哈哈

**Comment by Yuhsin000** [comment]
志同道合的人討論起來很開心，一起ㄎㄧㄤ

**Comment by PoooYen** [comment]
github就是牛，懂者自懂...

**Comment by d26488399** [comment]
Github果然是交友平台

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，原來Github還有這種功能。

**Comment by yenwei12138** [comment]
98%使用者為男性.....
這平台陽氣也太重了吧^^

**Comment by eason8479** [comment]
不是要貶低你學長的意思，不過git hub 上綠油油的一片某些時候也是刷出來的（改一行傳一次之類的）。

重要的應該還是要看那些修改到底是改了哪些東西，而不是一味地追求數量。

**Comment by ching020721** [comment]
github不只是工程師交流地，也是很好的社交平台阿哈哈

**Comment by Chiway1229** [comment]
超好笑 裡面的工程師都腦洞大開

**Comment by Kotori0501** [comment]
哇！原來GitHub有這麼多功能
我還記得開學第一週在python課上使用時，還在研究GitHub到底怎麼用

**Comment by dabbie109303501** [comment]
原來github是工程師的交友平台，真的漲姿勢ㄌw
